src/foam/u2/view/LazyScrollManager.js

```diff
@@ -379,6 +379,7 @@
         var limit = ( this.data && this.data.limit_ ) || undefined;
         return this.data$proxy.select(this.Count.create()).then(s => {
           this.daoCount = limit && limit < s.value ? limit : s.value;
+          this.data$proxy.cmd_(x, foam.dao.DAO.PURGE_CMD);
```

Comment:
Can you comment why this is needed.

Reply:
we need to rebuild the cache because the new item added might fall within the range of index already in the query caching dao so that when we do a select for the missing range, we return a duplicated element

Reply:
I meant "Can you add a comment...".